src/chi/components/customize-panel/customize-panel.scss

```
@@ -0,0 +1,66 @@
+.chi-customize-panel {
+  .chi-drawer {
+    .chi-drawer__content {
+      border-top: $customize-panel-border;
+      padding-top: $customize-panel-content-padding;
+      padding-bottom: $customize-panel-content-padding;
+      flex-grow: 0;
+
+      .chi-search__input {
+        margin-bottom: $customize-panel-search-margin-bottom;
+      }
+    }
+
+    .chi-customize-panel__items-header {
+      display: flex;
+      justify-content: space-between;
+
+      > span {
+        font-weight: $font-weight-semi-bold;
+      }
+
+      &.chi-customize-panel__header-sort {
+        align-items: center;
+        padding-bottom: $customize-panel-items-header-sort-padding-bottom;
+      }
+
+      &.chi-customize-panel__header-add {
+        flex-direction: column;
+        padding-bottom: $customize-panel-content-padding;
+        border-bottom: $customize-panel-border;
```

Names should match the attribue

```suggestion
        border-bottom: $customize-panel-border-bottom;
```